Apps/Playground/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ set(BABYLONSCRIPTS
     "../BabylonScripts/babylon.glTF2FileLoader.js"
     "../BabylonScripts/babylon.max.js"
     "../BabylonScripts/babylon.max.js.map"
+    "../BabylonScripts/babylon.module.d.ts"
```

Comment:
Thanks for adding this. There are more `d.ts` files though (e.g. for the glTF loader). We can add this later though.

Reply:
Yeah I think we should also bring in the corresponding map files in a separate PR.